### PR TITLE
Fix ClipPlane.jsx

### PR DESCRIPTION
**Pull request by Niels-IO**
Fixes the Clipping Plane crash

### CODE REVIEW

**Comment by Niels-IO** [comment]
Should fix #102

File: packages/studio/src/ClipPlane.jsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export default function ClipPlane({ clippingPlane, sideColor, children }) {
       node.material = backFaceStencilMat;
     });
 
-    const planeGeom = new THREE.PlaneBufferGeometry();
+    const planeGeom = new THREE.PlaneGeometry();
```
Comment (sgenoud) [inline packages/studio/src/ClipPlane.jsx:98]: Can you explain me what was going on? (I just want to learn - I have no idea why one is better than the other).

Comment (sgenoud) [reply packages/studio/src/ClipPlane.jsx:98]: BTW, thanks a lot for the PR - the first non typo code PR for replicad!

Comment (Niels-IO) [reply packages/studio/src/ClipPlane.jsx:98]: Hi @sgenoud, this StackOverflow answer explains it in detail:
https://stackoverflow.com/a/76792805/17515424
It seems they just removed it from Three at some point.

By the way, it is such an awesome library that you created here. Very well done!

**Pull request merged**